reconcile/slack_usergroups.py

```diff
@@ -270,6 +282,20 @@ def get_slack_usernames_from_owners(owners_from_repo, users, usergroup):
     return all_slack_usernames
 
 
+def get_slack_usernames_from_schedule(schedule):
+    if schedule is None:
+        return []
+    now = datetime.utcnow()
+    all_slack_usernames = []
```

Comment:
This one is probably better served as a `set`, I can imagine there being duplicates.